packages/example-react-ts/stories/Button.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentMeta, ComponentStoryObj } from '@storybook/react';
+import { Button } from './Button';
+
+export default {
+  title: 'Example/Button',
+  component: Button,
+  argTypes: {
+    backgroundColor: { control: 'color' },
```

Comment:
If we add a `.storybook/preview.ts` with the contents below, we can remove this `argTypes`, and the actions addon will wire up to any `onX` methods.

```
export const parameters = {
  actions: { argTypesRegex: '^on[A-Z].*' },
  controls: {
    matchers: {
      color: /(background|color)$/i,
      date: /Date$/,
    },
  },
};
```